test/override.js

```diff
@@ -33,6 +33,24 @@ describe('@override@', function () {
       done();
     });
   });
+  it("@builders@ overrides tgtBrowser abstraction", function (done) {
```

Comment:
Since the change affects both serverUrl and tgtBrowser, wouldn't it make sense to have a test that verifies both? or two tests?